core/lib/dal/src/eth_sender_dal.rs

```diff
@@ -986,9 +993,16 @@ impl EthSenderDal<'_, '_> {
         .unwrap()
         .unwrap();
         let eth_tx_id = match op_type {
-            AggregatedActionType::Commit => row.eth_commit_tx_id,
-            AggregatedActionType::PublishProofOnchain => row.eth_prove_tx_id,
-            AggregatedActionType::Execute => row.eth_execute_tx_id,
+            AggregatedActionType::L1Batch(L1BatchAggregatedActionType::Commit) => {
+                row.eth_commit_tx_id
+            }
+            AggregatedActionType::L1Batch(L1BatchAggregatedActionType::PublishProofOnchain) => {
+                row.eth_prove_tx_id
+            }
+            AggregatedActionType::L1Batch(L1BatchAggregatedActionType::Execute) => {
+                row.eth_execute_tx_id
+            }
+            _ => panic!("Unsupported"),
```

Comment:
nit: you can accept `L1BatchAggregatedActionType` as input and do not panic in the method